.github/workflows/ci.yml

```diff
@@ -69,34 +69,37 @@ jobs:
       - uses: actions/checkout@v3
       - name: Set up Native Dependencies
         uses: ./.github/actions/setup-native-dep
+        id: native-deps
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: stable
       - uses: Swatinem/rust-cache@v2
-      - run: cargo test
+      - run: cargo test --features testing
+        env:
+          CAIRO_NATIVE_RUNTIME_LIBDIR: ${{ steps.native-deps.outputs.cairo-native-runtime-libdir }}
+          MLIR_SYS_170_PREFIX: /usr/lib/llvm-17/
+          LLVM_SYS_170_PREFIX: /usr/lib/llvm-17/
+          TABLEGEN_170_PREFIX: /usr/lib/llvm-17/
```

Comment:
This env vars are already available globally, do they need to be set up again?

Reply:
yes, idk why but they're not working when declared globally, I will remove the global ones